examples/stable_diffusion/README.md

```diff
@@ -4,9 +4,9 @@
 
 2. Run: `git clone https://github.com/skypilot-org/skypilot.git && cd examples/stable_diffusion`.
 
-3. Run `sky launch -c stable-diffusion stable_diffusion_docker.yaml`.
+3. Run `sky launch -c stableDiffusion faster_stable_diffusion_docker.yaml`  
```

Comment:
Any thought on just overwriting the original yaml, vs. introducing a new file?

Reply:
Sure we can overwrite existing yaml file.